Sources/AWSLambdaRuntimeCore/LambdaContext.swift

```diff
@@ -68,11 +70,17 @@ extension Lambda {
         /// For invocations from the AWS Mobile SDK, data about the client application and device.
         public let clientContext: String?
 
+        /// Context baggage.
+        public var baggage: BaggageContext
```

Comment:
👍

Reply:
Oh actually, `Lambda.Context` is a class... so this may need lock protection

Reply:
made it thread safe but I think it needs to be revisited https://github.com/swift-server/swift-aws-lambda-runtime/pull/162#discussion_r471457430